hal/api/InterruptManager.h

```diff
@@ -114,8 +114,8 @@ class InterruptManager {
     bool remove_handler(pFunctionPointer_t handler, IRQn_Type irq);
 
 protected:
-    virtual void lock();
-    virtual void unlock();
+    void lock();
```

Comment:
not relevant to this changeset but it is touched, so I ask: why these 2 are protected and the rest of lock/unlock are public? why not private/